smtk/utils/helpers.py

```diff
@@ -24,3 +24,26 @@ def twitter_auth(consumer_key=None, consumer_secret=None, access_key=None, acces
             return None
 
     return auth
+
+
+def facebook_auth(app_id=None, app_secret=None):
+    """Validate and create facebook access token"""
+
+    if not app_id:
+        app_id = os.environ.get('FB_APP_ID', app_id)
+    if not app_secret:
+        app_secret = os.environ.get('FB_APP_SECRET', app_secret)
+
+    auth = [app_id, app_secret]
+
+    for credential in auth:
+        if not isinstance(credential, str):
+            print('credential must be string')
+            return None
+        if len(credential) <= 0:
+            print('invalid credential {}'.format(credential))
+            return None
+
+    access_token = auth[0] + "|" + auth[1]
```

Comment:
I'm not as familiar with the facebook API but since I believe we should (probably) use facepy. The token generation for facepy takes a list so I think would want our facebook "auth" return object to be a list. What do you think? 
`[app_id, app_secret, api_version]`
```
auth_token = facepy.utils.get_application_access_token(
        app_id, app_secret, api_version='2.6')
graph = facepy.GraphAPI(auth_token)
```
The way I implemented it for twitter was to return the arguments that gets passed to python-twitter.

Ex:
```
self.auth = helpers.twitter_auth()
self.api = twitter.Api(*self.auth)
```